### PR TITLE
New_devsurvey

### PR DESCRIPTION
**Pull request by Mankavelda**
Feat: new devsurvey with vue js
this branch contains the new developers survey redesigned with vue js.
header section:
Screenshot from 2022-09-21 09-02-51

### CODE REVIEW

File: new_devsurvey/src/components/HeaderComponent.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+				</div>
+
+				<div class="d-flex justify-content-center mb-5">
+					<a href="https://opencollective.com/osscameroon/donate" target="_blank">
+						<img src="https://opencollective.com/webpack/donate/button@2x.png?color=white" width=200 />
+					</a>
+				</div>
+
+				<div class="row d-flex justify-content-center text-secondary">
+					<p style="text-align: center">You are a cameroonian developer, please take this 3 minutes survey <a
```
Comment (Asam237) [inline new_devsurvey/src/components/HeaderComponent.vue:18]: We just need to increase the police

File: new_devsurvey/src/components/TableComponent.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<template>
```
Comment (Asam237) [inline new_devsurvey/src/components/TableComponent.vue:1]: This component is unfinished

**Review by Asam237 — CHANGES_REQUESTED** [review]

File: new_devsurvey/src/components/TableComponent.vue (hunk 2 above)
Comment (Mankavelda) [reply new_devsurvey/src/components/TableComponent.vue:1]: Please explain better

File: new_devsurvey/src/components/HeaderComponent.vue (hunk 1 above)
Comment (Asam237) [reply new_devsurvey/src/components/HeaderComponent.vue:18]: The best thing would be to finish a component before pushing, but here, it seems that the diagram has not been completed

File: new_devsurvey/src/components/TableComponent.vue (hunk 2 above)
Comment (Asam237) [reply new_devsurvey/src/components/TableComponent.vue:1]: The best thing would be to finish a component before pushing, but here, it seems that the diagram has not been completed

**Comment by Asam237** [comment]
Conflicts must be solved too!

**Comment by Asam237** [comment]
I can't run the project.
However, I deleted the node_module folder and then I typed <yarn> again but I still have this error

Screenshot from 2022-09-20 16-32-02

**Comment by Mankavelda** [comment]
Hello @Asam237  please try "npm run serve"

> I can't run the project. However, I deleted the node_module folder and then I typed again but I still have this error
> 
> Screenshot from 2022-09-20 16-32-02

**Comment by Mankavelda** [comment]
Hello @Asam237  please run by "npm run serve"

**Comment by Asam237** [comment]
To solve the problem you would need to install this library: npm install @popperjs/core --save

**Pull request closed without merging**